frontend/src/actions/TableActions.js

```diff
@@ -499,8 +500,16 @@ function createCollapsedRows({
  *
  * @param {string} tableId
  * @param {array} rows
+ * @param {array} changedIds - array with ids of the rows to be changed that are sent with a websocket message
```

Comment:
(loud thinking) maybe you should consider a more intention revealing name to that `changedIds` param.

Maybe something like `preserveCollapsedStateToRowIds`.

If the name i gave is also wrong because that's not what it does or you have a better solution,
feel free to come with a better name.